src/components/logview/logview.tsx

```diff
@@ -8,7 +8,6 @@ enum LogDataType {
 }
 interface LogData {
   type: LogDataType
-  url?: string
```

Comment:
I wanted to standarize the LogData type, there's no reason to differentiate between `url` and `title` property, they both act as titles.